### PR TITLE
chore(e2e): Remove flipper and fix verify user tests

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Description of changes

This change:
* fixes android workflow - unblock CI
* removes Flipper dependency from ios example app to speed up the build (flipper is a debugging tool that we don't use, since we tend to use react-native debugging tools, so disabled it overall, not just in CI).
* fix test id suffix for authenticator fields

#### Description of how you validated changes

Modified `test-internal-prs` reference to `reusable-e2e`  then reverted that change.
Result: https://github.com/aws-amplify/amplify-ui/actions/runs/4967755723/jobs/8890077669

Build time for ios reduced to [8min](https://github.com/aws-amplify/amplify-ui/actions/runs/4964073405/jobs/8884036244) from an average of [15m](https://github.com/aws-amplify/amplify-ui/actions/runs/4952570949/jobs/8859413024).
Overall iOS tests now take ~40 min, an improvement of around 35%.

Android is still taking significantly longer, typing in input fields is in slow motion even with disabled synchronization. 

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by zchenwei — COMMENTED** [review]
Thanks for making the fixes! The e2e android workflow is still using the commit from main. One little trick is change `@main` to your branch in [this line](https://github.com/aws-amplify/amplify-ui/blob/main/.github/workflows/test-internal-prs.yml#L140) and push to kick off a run to see if anything else needs to be updated but otherwise looks good to me!

**Comment by ioanabrooks** [comment]
> Thanks for making the fixes! The e2e android workflow is still using the commit from main. One little trick is change `@main` to your branch in [this line](https://github.com/aws-amplify/amplify-ui/blob/main/.github/workflows/test-internal-prs.yml#L140) and push to kick off a run to see if anything else needs to be updated but otherwise looks good to me!

Done! https://github.com/aws-amplify/amplify-ui/actions/runs/4967755723/jobs/8890077669

**Review by zchenwei — APPROVED** [review]
LGTM :shipit:

File: examples/react-native/ios/Podfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ target 'ReactNative' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable the next line.
-  use_flipper!()
+  # use_flipper!()
```
Comment (calebpollman) [inline examples/react-native/ios/Podfile:31]:
```suggestion
  # NOTE: flipper disabled by default to improve iOS build times for CI
  # use_flipper!()
```

**Review by calebpollman — APPROVED** [review]
LGTM, just left a suggestion adding a comment on the flipper disabling

File: examples/react-native/ios/Podfile (hunk 1 above)
Comment (ioanabrooks) [reply examples/react-native/ios/Podfile:31]: Thanks, good call, will include this in my next PR, want to unblock CI

**Pull request merged**